files/default/sql/ddl/updates/3.2.0.sql

```diff
@@ -136,3 +136,14 @@ CREATE TABLE IF NOT EXISTS `job_schedule` (
     UNIQUE KEY `job_id` (`job_id`),
     CONSTRAINT `fk_schedule_job` FOREIGN KEY (`job_id`) REFERENCES `jobs` (`id`) ON DELETE CASCADE
     ) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
+-- alert changes for feature monitoring
+ALTER TABLE `feature_group_alert`
+    MODIFY `feature_group_id` int NULL,
+    ADD `feature_view_id` int NULL,
+    ADD `entity_type` int NOT NULL,
```

Comment:
You need to provide a default here for things that already exist in the db

Reply:
added default for entity type. For other FG id and FV id they are foreign keys, dont think we need anything there.

Reply:
Tested update DDL with data in table, works as expected